File: https:/github.com/lucius-lt/leetcode.git/58-LengthOfLastWord/58-LengthOfLastWord.c
// Last updated: 7/16/2026, 1:12:09 AM
int lengthOfLastWord(char* s) {
 
    int len = strlen(s);
    int i = len - 1;

    while (i >= 0 && s[i] == ' ') {
        i--;
    }
    int word_len = 0;
    while (i >= 0 && s[i] != ' ') {
        word_len++;
        i--;
    }

    return word_len;
}

File: https:/github.com/lucius-lt/leetcode.git/58-LengthOfLastWord/58-LengthOfLastWord.java
// Last updated: 7/16/2026, 1:12:03 AM
class Solution {
    public int lengthOfLastWord(String s) {
        int i = s.length() - 1, count = 0;

        while (i >= 0 && s.charAt(i) == ' ') i--;

        while (i >= 0 && s.charAt(i) != ' ') {
            count++;
            i--;
        }

        return count;
    }
}
